From f163e1dede72014608226a718ade142634e28458 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Mon, 7 Mar 2016 20:44:25 +0100
Subject: [PATCH] man: ip-xfrm.8: Document missing parameters

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1251186
Upstream Status: iproute2.git commit a7eef7aa70b2d

commit a7eef7aa70b2d7ea2a78841e1cc188fa416e0f7f
Author: Phil Sutter <phil@nwl.cc>
Date:   Wed Mar 2 19:20:05 2016 +0100

    man: ip-xfrm.8: Document missing parameters

    Namely, 'extra-flag' of 'ip xfrm state' and 'flag' of 'ip xfrm policy'.

    Signed-off-by: Phil Sutter <phil@nwl.cc>
---
 man/man8/ip-xfrm.8 | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/man/man8/ip-xfrm.8 b/man/man8/ip-xfrm.8
index aea4fda..a9c5890 100644
--- a/man/man8/ip-xfrm.8
+++ b/man/man8/ip-xfrm.8
@@ -53,6 +53,8 @@ ip-xfrm \- transform configuration
 .IR ADDR "[/" PLEN "] ]"
 .RB "[ " ctx
 .IR CTX " ]"
+.RB "[ " extra-flag
+.IR EXTRA-FLAG-LIST " ]"
 
 .ti -8
 .B "ip xfrm state allocspi"
@@ -191,6 +193,13 @@ ip-xfrm \- transform configuration
 .IR SPORT " " DPORT " " OADDR
 
 .ti -8
+.IR EXTRA-FLAG-LIST " := [ " EXTRA-FLAG-LIST " ] " EXTRA-FLAG
+
+.ti -8
+.IR EXTRA-FLAG " := "
+.B dont-encap-dscp
+
+.ti -8
 .BR "ip xfrm policy" " { " add " | " update " }"
 .I SELECTOR
 .B dir
@@ -242,6 +251,8 @@ ip-xfrm \- transform configuration
 .IR ACTION " ]"
 .RB "[ " priority
 .IR PRIORITY " ]"
+.RB "[ " flag
+.IR FLAG-LIST "]"
 
 .ti -8
 .B "ip xfrm policy flush"
-- 
1.8.3.1

